include/openmc/math_functions.h

```diff
@@ -161,6 +161,38 @@ extern "C" double maxwell_spectrum(double T);
 
 extern "C" double watt_spectrum(double a, double b);
 
+//==============================================================================
+//! Samples an energy from the Gaussian energy-dependent fission distribution.
+//!
+//! Samples from a Gaussian spectrum with a given mean and standard deviation
+//! The PDF is defined as s(x) = (1/sigma*sqrt(2) * e-((mu-x)/sigma)^2
+//! Its sampled according to 
+//! http://www-pdg.lbl.gov/2009/reviews/rpp2009-rev-monte-carlo-techniques.pdf
+//! section 33.4.4
+//!
+//! @param mean mean of the Gaussian distribution
+//! @param std_dev standard deviation of the Gaussian distribution
+//! @result The sampled outgoing energy
+//==============================================================================
+
+extern "C" double gaussian_spectrum(double mean, double std_dev);
```

Comment:
Maybe just `normal_variate`. This could be used generically (i.e., outside the context of an energy "spectrum").